python-sdk/nuscenes/map_expansion/arcline_path_utils.py

```diff
@@ -4,6 +4,9 @@
 from typing import Dict, Any, List, Tuple
 import math
 
+Pose = Tuple[float, float, float]
```

Comment:
@freddyaboulton Final comment on this PR. Can you please add a comment with (x, y, yaw) for pose?